test/passing/comments_args.ml.ref

```diff
@@ -5,7 +5,9 @@ let emit_wrapper_function =
     (Hhas_pos.pos_to_span ast_fun.Ast.f_span)
     false (* is_async *) false (* is_generator *) false
     (* is_pair_generator *) hoisted true (* no_injection *) true
-    (* inout_wrapper *) is_interceptable false (* is_memoize_impl *) Rx.NonRx
+    (* inout_wrapper *) is_interceptable false
+    (* is_memoize_impl *)
+    Rx.NonRx
```

Comment:
closer to the input, probably partially due to the comment attachment (will be improved to another PR)